src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String output = readFile(fromFileName);
+        writeToFile(output, toFileName);
+    }
+
+    private String readFile(String sourceFile) {
+        File file = new File(sourceFile);
+        int supplyResult = 0;
+        int buyResult = 0;
+        int result = 0;
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
```

Comment:
[ if you are using classes that implement an AutoCloseable interface, we should use it with try-with-resources.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#try-with-resources)